src/components/category.js

```diff
@@ -192,19 +192,7 @@ export default class Category extends React.Component {
         {emojis &&
           !emojis.length && (
             <div>
-              <div>
-                {NimbleEmoji({
-                  data: this.data,
-                  ...emojiProps,
-                  size: 38,
-                  emoji: 'sleuth_or_spy',
-                  onOver: null,
-                  onLeave: null,
-                  onClick: null,
-                })}
-              </div>
-
-              <div className="emoji-mart-no-results-label">{i18n.notfound}</div>
+              <NotFound {...this.props} />
```

Comment:
Let's be explicit about the props we're passing to `NotFound`, not just passing them all in.